vuu-ui/showcase/src/examples/Apps/NewTheme.examples.tsx

```diff
@@ -116,7 +116,7 @@ const ShellWithNewTheme = () => {
   const { saveLayout } = useLayoutManager();
 
   const handleSave = useCallback(
-    (layoutMetadata: Omit<LayoutMetadata, "id">) => {
+    (layoutMetadata: Omit<LayoutMetadata, "id" | "created">) => {
```

Comment:
The fact we're having to do this omission of `id` and `created` anywhere we're sending a POST/PUT request makes me wonder whether we should have client-side DTOs as well. Not really a priority though, so unless it's a quick refactor it's not worth doing.

Reply:
good point. I ended up using `Omit<LayoutMetadata, "id" | "created">` in a bunch of places and we might as well have it as its own type

Reply:
added a LayoutMetadataDto type for this